### DISCUSSION TITLE
Possible to support esp32

### DISCUSSION THREAD

**Discussion by allbeq** [softwarecrash/Solar2MQTT-ESP8266#10] [General]
I have try to adapt code for esp32, replace libs:
#if defined(ESP8266)
#include <ESP8266WiFi.h>
#include <ESP8266mDNS.h>
#include <ESPAsyncTCP.h>
#endif

#if defined(ESP32)
#include <WiFi.h>
#include <ESPmDNS.h>
#include <AsyncTCP.h>
#endif

but have some issues..
Maybe did you have esp32 code?

**Comment by softwarecrash** [comment]
i dont have code for esp32, why use esp2, a cheap esp82 have enugh power for this job.
its not so easy as replace librays

**Comment by v-moto** [comment]
Maybe you can adapt the code for wemos s2 mini (esp32)
Because I accidentally ordered a pack of s2 instead of d1 and now I don't know what to do with them
I tried to adapt it myself, but my knowledge was not enough
If you manage to do it, the project will be donated)

**Reply by softwarecrash** [reply]
adapting for esp32 is not planed yet
